lib/ansible/modules/cloud/amazon/dynamodb.py

```diff
@@ -0,0 +1,359 @@
+
+# Creates a single item in 'narcos' DynamoDB table where column 'bank' equals 'hsbc' string,
+# column 'quantity' equals '1000' numeric and column 'person' equals 'ochoa' string
+- name: Creates a new record
+  dynamodb:
+    table: narcos
+    action: put
+    item: {"bank": {"S": "hsbc"},"quantity": {"N": "1000"},"person": {"S": "ochoa"}}
+
+# Updates the 'quantity' attibute value from a single item.
+- name: Updates arribute 'person'
+  dynamodb:
+    table: narcos
+    action: update
+    primary_key: {"bank": {"S": "hsbc"}}
+    update_expression: 'SET quantity = :number'
+    expression_attribute_values: {":number": {"N": "2000"}}
+
+# Updates the 'status' (dynamodb protected keyword) attibute value from a single item.
+- name: Updates arribute 'status'
+  dynamodb:
+    table: narcos
+    action: update
+    primary_key: {"bank": {"S": "hsbc"}}
+    update_expression: 'SET #s = :number'
+    expression_attribute_values: {":number": {"N": "2000"}}
+    expression_attribute_names: {"#s" : "status"}
+
+# Deletes the 'person' attibute value from a single item. The table has a composite
+# primary key. 'bank' is the primary key and 'quantity' is the sort key.
+- name: Deletes arribute 'person'
+  dynamodb:
+    table: narcos
+    action: update
+    primary_key: {"bank": {"S": "hsbc"}, "quantity": {"N": "1000"}}
+    update_expression: 'REMOVE person'
+
+# Connects with awscli profile 'development' and
+# deletes a single item from 'releases' DynamoDB table
+# where the column 'project' equals string 'potatoes' (also primary key)
+# and column 'version' equals numeric '123456'
+- name: Delete a single item
+  dynamodb:
+    profile: development
+    table: releases
+    action: delete
+    primary_key: {"project": {"S": "potatoes"}}
+    condition_expression: version = :number
+    expression_attribute_values: {":number": {"N": "123456"}}
+'''
+
+RETURN = '''
+item:
+    description: Item when you peform a 'get' action.
+    returned: success
+    type: dict
+    sample:
+        {
+            "bank": {
+                "S": "hsbc"
+            },
+            "quantity": {
+                "N": "1000"
+            },
+            "person": {
+                "S": "ochoa"
+            }
+        }
+'''
+
+import ansible.module_utils.ec2
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.ec2 import boto_exception
+
+
+try:
+    import boto3
+    import botocore
+    HAS_BOTO3 = True
+except ImportError:
+    HAS_BOTO3 = False
+
+
+def get(connection, table, primary_key, projection_expression, result):
+
+    if projection_expression:
+        response = connection.get_item(
+            TableName=table,
+            Key=primary_key,
+            ProjectionExpression=projection_expression
+        )
+    else:
+        response = connection.get_item(
+            TableName=table,
+            Key=primary_key
+        )
+
+    if 'Item' in response:
+        result['item'] = response['Item']
```

Comment:
Is anything else returned in response? If so, it should probably be returned by the module. Likewise with the other functions, where you're currently throwing away the entire boto3 result. See https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/amazon/GUIDELINES.md#returning-values